website/docs/addon_motionbuilder_admin.md

```diff
@@ -10,6 +10,27 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 {versions.MotionBuilder_Badge}
 </ReactMarkdown>
 
-:::note Work in progress
-This part of documentation is still work in progress.
+:::caution Unknown support for Motion Builder 2023 and below
+
+The AYON Motion Builder addon was initially developed with Motion Builder 2024 and 2025.
```

Comment:
Looking at google results the application is referred to as MotionBuilder. So we we may want to update anywhere we mention "Motion Builder" to that?
```suggestion
The AYON MotionBuilder addon was initially developed with MotionBuilder 2024 and 2025.
```